test/ruby/test_process.rb

```diff
@@ -1424,6 +1424,11 @@ def test_argv0_keep_alive
     REPRO
   end
 
+  def test_argv0_frozen
+    assert_predicate Process.argv0, :frozen?
+    assert_predicate $0, :frozen?
+  end
```

Comment:
Probably doesn't matter much, but that's technically a behavior change, right?

Reply:
Yes, bug fixes change behaviors.